InternshipBE/WebApi/Models/Authorization/AuthOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WebApi.Models.Authorization
+{
+	public class AuthOptions
+	{
+		public const string Issuer = "ExadelAuthServer";
+
+		public const string Audience = "ExadelTeam";
```

Comment:
If it is necessary will be better move it to web.config where we can change it without rebuild app